### PR TITLE
SPOP with COUNT returns empty list (not nil)

### PR DESCRIPTION
**Pull request by sazzad16**
Related code: https://github.com/redis/redis/blob/f5235b2d7660b163c739325ec9284f97d5b963be/src/t_set.c#L465-L466

### CODE REVIEW

File: commands/spop.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ When called without the `count` argument:
 
 When called with the `count` argument:
 
-@array-reply: the removed members, or `nil` when `key` does not exist.
+@array-reply: the removed members, or an empty set when `key` does not exist.
```
Comment (itamarhaber) [inline commands/spop.md:17]:
```suggestion
@array-reply: the removed members, or an empty array when `key` does not exist.
```

**Review by itamarhaber — APPROVED** [review]

**Comment by itamarhaber** [comment]
Thanks @sazzad16 - changed "set" to "array" for consistency with other commands' replies.

**Comment by sazzad16** [comment]
@itamarhaber I don't as long as it says empty, not nil.

**Comment by sazzad16** [comment]
@itamarhaber I don't mind as long as it says empty, not nil.

**Pull request merged**